### PR TITLE
Refactor/unit test/account -covered more unit tests

### PR DESCRIPTION
**Pull request by mahabaleshwarhnr**
covers more test on following 
- Create account
- User Account
- Launcher activity
- logout

### CODE REVIEW

**Review by KrupaDesai22 — APPROVED** [review]

File: user/account/src/main/java/in/projecteka/jataayu/user/account/viewmodel/UserAccountsViewModel.kt [hunk 1]
```diff
@@ -45,7 +47,10 @@ class UserAccountsViewModel(private val repository: UserAccountsRepository,
                     userAccountLoading.set(it.isLoading)
                     isCurrentlyFetching()
                 }
-                is Success -> updatePatient(it.data?.linkedPatient)
```
Comment (AnusreeSajeevan) [inline user/account/src/main/java/in/projecteka/jataayu/user/account/viewmodel/UserAccountsViewModel.kt:48]: what is the need of this change?

File: user/account/src/test/java/in/projecteka/jataayu/user/account/viewmodel/UserAccountsViewModelTest.kt [hunk 2]
```diff
@@ -93,6 +100,42 @@ class UserAccountsViewModelTest {
...
+//                callback.onResponse(profileResponseCall, Response.success(linkedAccountsResponse))
+//            }
+//        viewModel.getUserAccounts()
+//        Mockito.verify(repository).getUserAccounts()
+//        Mockito.verify(profileResponseCall).enqueue(ArgumentMatchers.any())
+//        Assert.assertEquals(linkedAccountsResponse, viewModel.linkedAccountsResponse.value)
+//    }
+
+//    @Test
+//    fun shouldGetAccountsToDisplay() {
```
Comment (AnusreeSajeevan) [inline user/account/src/test/java/in/projecteka/jataayu/user/account/viewmodel/UserAccountsViewModelTest.kt]: remove commented code. comment with "//TODO: Modify after refactor" if this test will be added in the future.

Apply this for all commented test cases.

Comment (mahabaleshwarhnr) [reply user/account/src/test/java/in/projecteka/jataayu/user/account/viewmodel/UserAccountsViewModelTest.kt]: all commented code and TODO are already removed in the latest commit. Can you check this 70f362a commit

File: user/account/src/main/java/in/projecteka/jataayu/user/account/viewmodel/UserAccountsViewModel.kt (hunk 1 above)
Comment (mahabaleshwarhnr) [reply user/account/src/main/java/in/projecteka/jataayu/user/account/viewmodel/UserAccountsViewModel.kt:48]: this code is not removed. few lines are added or removed.so showing like that

**Review by AnusreeSajeevan — APPROVED** [review]

**Pull request merged**